packages/readme.md

```diff
@@ -0,0 +1,233 @@
+# Structure of package.mk files
+
+## Introduction
+
+The package.mk file defines variables and functions to build a package.
+
+## Variables
+All avialible variable, to control the build behavoir of your package.
+Please use these in the order, listed here.
+
+#### Base
+
+| Variable    | Default | Required |Description |
+|-------------|---------|----------|------------|
+| PKG_NAME    | -       | yes | Name of the packaged software application. Should be lowercase |
+| PKG_VERSION | -       | yes | Version of the packaged software application |
+| PKG_SHA256  | -       | yes | SHA256 hashsum of the application download file |
+| PKG_ARCH    | any     | no  | Architectures for which the package builds. `any` or a space separated list of `aarch64`, `arm` or `x86_64` |
+| PKG_LICENSE | -       | yes | License of the software application. [Reference](https://github.com/LibreELEC/LibreELEC.tv/tree/master/licenses) |
```

Comment:
It needs to be agreed in the team, but I think we should use SPDX notation for this